src/program.ts

```diff
@@ -28,6 +29,11 @@ program
     'Style of generated tree ("arrows" | "pipes")',
     STYLE
   )
+  .option(
+    '-c, --copy',
```

Comment:
The program will take the full form of the option and assign it to the options object, for example here it would be `options.copy`, if we define it as `--auto-copy` it will be accessible with `options.autoCopy`. This leads me to my commend below 👇